src/main/java/com/bookstore/dto/CreateBookRequestDto.java

```diff
@@ -1,13 +1,22 @@
 package com.bookstore.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotNull;
 import java.math.BigDecimal;
 import lombok.Data;
+import org.hibernate.validator.constraints.ISBN;
 
 @Data
 public class CreateBookRequestDto {
+    @NotNull
     private String title;
```

Comment:
I recommend also checking for empty input and minimum and maximum string length. Do this with other fields where you think it's need.